### PR TITLE
[Docs] Add PR document

### PR DESCRIPTION
**Pull request by MeowZheng**
## Motivation

+ This document is to help users to contribute their work to OpenMMLab repos.

## Modification

+ Add pr.md in docs/community and its Chinese version in docs_zh_CN/community.

### CODE REVIEW

**Comment by zhouzaida** [comment]
please fix the lint

File: docs/community/pr.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+## PR
```
Comment (innerlee) [inline docs/community/pr.md:1]: make title explicit

```markdown
## Pull Request (PR)
```

File: docs/community/pr.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+## PR
+
+### What is PR
+`PR` is the abbreviation of `Pull Request`. Here's the definition of `PR` in the [official document](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) of Github.
+
+> Pull requests let you tell others about changes you've pushed to a branch in a repository on GitHub. Once a pull request is opened, you can discuss and review the potential changes with collaborators and add follow-up commits before your changes are merged into the base branch.
+
+### Basic Workflow
```
Comment (innerlee) [inline docs/community/pr.md:8]: Your First PR

Comment (innerlee) [reply docs/community/pr.md:8]: And add a section Your Second PR

File: docs/community/pr.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
+## PR
+
+### What is PR
+`PR` is the abbreviation of `Pull Request`. Here's the definition of `PR` in the [official document](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) of Github.
```
Comment (innerlee) [inline docs/community/pr.md:4]: insert blank line after title

**Comment by ZwwWayne** [comment]
Please make 3.png smaller

**Review by innerlee — APPROVED** [review]

**Review by zhouzaida — APPROVED** [review]
LGTM

**Pull request merged**